scripts/build-worker.sh

```diff
@@ -5,6 +5,16 @@ if [[ -n "${DEBUG-}" ]]; then
   set -x
 fi
 
+# Workaround until the artifact proxying service is set up properly.
+# At least this gives us terminal output
+function printLog() {
+    if [[ ${LOG_LOCATION-} ]]; then
+        cat "${LOG_LOCATION}"
```

Comment:
This may be over-thinking, but would it be better to send to `stderr`?

Reply:
Probably for correctness yes